wallet/src/de/schildbach/wallet/util/TransactionExtensions.kt

```diff
@@ -1,33 +0,0 @@
-package de.schildbach.wallet.util
-
-import de.schildbach.wallet.Constants
-import de.schildbach.wallet.WalletApplication
-import org.bitcoinj.core.Address
-import org.bitcoinj.core.Coin
-import org.bitcoinj.core.Transaction
-import org.bitcoinj.script.ScriptException
-import java.util.ArrayList
-
-fun Transaction.isOutgoing(): Boolean {
-    return getValue(WalletApplication.getInstance().wallet).signum() < 0
```

Comment:
An extension method with `WalletApplication.getInstance()` is not great - breaks DI and is hard to test. Better to call `getValue` with a provided `TransactionBag` instead.